src/applications/disability-benefits/new-686/config/chapters/report-child-stopped-attending-school/child-information/childInformation.js

```diff
@@ -0,0 +1,54 @@
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+
+import { isChapterFieldRequired } from '../../taskWizard/wizard/helpers';
+import { genericSchemas } from '../../../generic-schema';
+import { validateName } from '../../../utilities';
+
+const { fullName, date } = genericSchemas;
+
+export const schema = {
+  type: 'object',
+  properties: {
+    childNoLongerAtSchoolName: fullName,
+    dateChildLeftSchool: date,
+  },
+};
+
+export const uiSchema = {
+  childNoLongerAtSchoolName: {
+    'ui:validations': [validateName],
+    first: {
+      'ui:required': formData =>
```

Comment:
We decided to use `ui:required` instead of the `required` property on schema because a lot of these workflows are contingent on what a user will select to do at the very beginning of the form.